output/derived/wb_clean/sconscript.log

```diff
@@ -1,54 +1,2 @@
-*** Builder log created: {2022-05-12 12:47:06}
-*** Builder log completed: {2022-05-12 12:47:07}
-
-  ___  ____  ____  ____  ____ ®
- /__    /   ____/   /   ____/      17.0
-___/   /   /___/   /   /___/       MP—Parallel Edition
-
- Statistics and Data Science       Copyright 1985-2021 StataCorp LLC
-                                   StataCorp
-                                   4905 Lakeway Drive
-                                   College Station, Texas 77845 USA
-                                   800-STATA-PC        https://www.stata.com
-                                   979-696-4600        stata@stata.com
-
-Stata license: Unlimited-user 4-core network, expiring 17 Nov 2022
-Serial number: 501709306977
-  Licensed to: hbsuser
-               Harvard Business School
-
-Notes:
-      1. Stata is running in batch mode.
-      2. Unicode is supported; see help unicode_advice.
-      3. More than 2 billion observations are allowed; see help obs_advice.
-      4. Maximum number of variables is set to 5,000; see help set_maxvar.
-
-. do "source/derived/wb_clean/takelogs.do" 
-
-. version 14
-
-. set more off
-
-. preliminaries
-
-. 
-. program main
-  1. 
-.     import delimited "output/derived/wb_clean/gdp_education.csv", clear
-  2. 
-.     gen log_gdp_2010 = log(gdp_2010)
-  3.     gen log_education_exp_2010 = log(education_exp_2010)
-  4. 
-.     export delimited "output/derived/wb_clean/gdp_education_logs.csv"
-  5. 
-. end
-
-. 
-. main
-(3 vars, 264 obs)
-(12 missing values generated)
-(100 missing values generated)
-file output/derived/wb_clean/gdp_education_logs.csv saved
-
-. 
-end of do-file
+*** Builder log created: {2022-08-24 12:23:23}
+*** Builder log completed: {2022-08-24 12:23:32}
```

Comment:
Intentional that Stata log is not included here?

Reply:
This is just because the output files were already built when scons went to build it. When this happens and no changes were made, scons just moves on rather than rebuilding it. I removed the outputs in https://github.com/JMSLab/Template/pull/56/commits/9af7f8d289b1be1c3ea5bbdb698cf368deb73936 and reran the data and the log was restored.